yarn.lock

```diff
@@ -696,8 +696,8 @@ ajv@^6.1.0:
     uri-js "^3.0.2"
 
 ajv@^6.5.0:
-  version "6.5.1"
-  resolved "https://registry.yarnpkg.com/ajv/-/ajv-6.5.1.tgz#88ebc1263c7133937d108b80c5572e64e1d9322d"
+  version "6.5.2"
```

Comment:
Is that normal that there is a diff on the `yarn.lock`?

Reply:
This happened while updating my local environment. It might not hurt to have this patch update in there.